tools/hangar/src/compatibility-spy.test.ts

```diff
@@ -8,7 +8,7 @@ compatibilityTestFiles.forEach((wingFile) => {
   test(wingFile, async ({ expect }) => {
     // "@winglang/compatibility-spy" will be installed in the tmpDir
     const platforms = ["sim", "node_modules/@winglang/compatibility-spy/lib"];
-    const args = ["test", "-o", "out.json"];
+    const args = ["test", "-o", "out.json", "--no-stream"];
```

Comment:
Does it break if this flag isn't provided? Streaming logs should still work in a non-tty

Reply:
There's a check so the streaming should work in a non-tty - I can try removing it from this test